tags/cncf-tags.md

```diff
@@ -133,6 +125,48 @@ As a starting point let’s be inspired by CNCF OSS Projects and by K8s SIGs.  T
 * May either have no explicit roles or responsibilities, or formally assigned roles (see above).
 * May not create the impression that they have any authority or formal responsibilities in the TAG other than assigned roles.
 
+### TOC Liaisons
+
+TOC is the technical governing body of the CNCF. It’s important that we ensure a good flow of governance information and feedback loops to and from the  TAGs. With several TAGs, it can be hard to connect for community wide consensus therefore each group is assigned at least one TOC liaison.
+
+TAG leads may call on liaisons to act as a point of contact from the TOC, be an advisor for governance or community health matters, or help the TAG prioritize what they’re doing to be in line with TOC needs.
+
+Liaisons do not make decisions for the TAG or on behalf of the TOC.
+
+Liaisons are assigned based on their expertise, interest, and specialized experience a given TAG may need at the time of assignment (adjustments can be made, if needed). The TOC distributes TAG assignments as equal as possible. After each TOC election, liaisons are either rotated or new members will assume one of the departing members’ groups.
+
+#### Responsibilities
+
+- Streamline communications between TOC and TAG chairs, and helps TAG to set priorities.
+- Communicate activities of the TAG to TOC beyond the public monthly updates.
+- Support and guide growth and development of the TAG.
+- Attend TAG meetings, as needed/requested.
```

Comment:
- let's add something about TAG's working groups as well?
- Ensure TAG leadership is healthy and has a good ladder and includes mentorship?

Reply:
@dims thanks, addressed your feedback.

Reply:
thanks @nikhita

Reply:
Just a suggestion: I think it would be good for the TOC the do something like a quarterly (or different cadence) liaisons check in, to see if the liaisons are aligned with the tag and recommend/brainstorm how to keep things moving forward in the TAG.   This could also be a topic of discussion during the TOC-TAG meetings at KubeCon.

Reply:
This seems to be spread in the 'expectactions' below and the streamline of communications above? There is also a reference to '- Keep the TOC in the loop and keep track of trackable (i.e. not sensitive) work in public issues.'.

We could consider formalizing this in a periodic report by the liaisons to the TOC as suggested by @raravena80 .

Reply:
Added `Provide updates regarding TAG activities and potential areas of improvement to the TOC at least once a quarter.` PTAL.

Reply:
lgtm